csss-site/src/events/templates/events/frosh/2020/index.html

```diff
@@ -0,0 +1,126 @@
+
+    <!-- Custom styles for this template -->
+    <link href="{% static 'events_static/frosh/2020/landing_page/css/soon.css' %}" rel="stylesheet">
+
+    <!-- Overrides for everything -->
+    <link href="{% static 'events_static/frosh/2020/css/overrides.css' %}" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="assets/js/html5shiv.js"></script>
+      <script src="assets/js/respond.min.js"></script>
+    <![endif]-->
+
+    <link href='https://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
+
+</head>
+<!-- START BODY -->
+
+<body class="nomobile">
+
+    <!-- START HEADER -->
+    <section id="header">
+        <div class="container">
+            <header class="row">
+                <div class="col-xs-10 col-xs-offset-1">
+                    <!-- HEADLINE -->
+                    <h1 data-animated="GoIn">Computing Science Student Society Frosh 2020</h1>
+                </div>
+            </header>
+
+
+
+            <!-- LAYER OVER THE SLIDER TO MAKE THE WHITE TEXT READABLE -->
+            <!--<div id="layer"></div>-->
+            <!-- END LAYER -->
+            <!-- START SLIDER -->
+            <!-- Slider Commented Out
+        <div class="row">
+            <div id="slider" class="col-lg-4 col-lg-offset-4 rev_slider">
+                <ul>
+                    <li data-transition="slideleft" data-slotamount="1" data-thumb="assets/frosh/new_header.png">
+                        <img src="assets/img/frosh/new/blueDisk.jpg">
+                    </li>
+                    <li data-transition="slideleft" data-slotamount="1" data-thumb="assets/img/frosh/bedrace.png">
+                        <img src="assets/img/frosh/frosh-slide-4.jpg">
+                    </li>
+                    <li data-transition="slideleft" data-slotamount="1" data-thumb="assets/img/frosh/Club_Night.JPG">
+                        <img src="assets/img/frosh/frosh-slide-3.jpg">
+                    </li>
+                </ul>
+            </div>
+        </div> -->
+            <!-- END SLIDER -->
+
+            <!-- START TIMER -->
+
+            <div class="row">
+                <img class="img-responsive center-block" src="{% static 'events_static/frosh/2020/images/TitleLogo_Green.png' %}">
+            </div>
+
+            <div class="row">
+                <div id="timer" data-animated="FadeIn">
+                    <div id="days" class="timer_box"></div>
+                    <div id="hours" class="timer_box"></div>
+                    <div id="minutes" class="timer_box"></div>
+                    <div id="seconds" class="timer_box"></div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-lg-4 col-lg-offset-4 mt centered">

```

Comment:
think you messed something up cause I get this error when clicking the button 
![Screenshot from 2020-08-25 17-17-39](https://user-images.githubusercontent.com/14242088/91240500-0f866d00-e6f7-11ea-9091-eaecf298c796.png)